templates/gcp/fips.html

```diff
@@ -0,0 +1,408 @@
+            <td data-heading="FIPS certified component">Linux kernel (generic)</td>
+            <td data-heading="Description">The Linux kernel cryptographic library</td>
+            <td class="u-align--right" data-heading="Version">5.4.0-1021</td>
+            <td class="u-align--right" data-heading="CMVP Certificate"><a href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4127">4127</a></td>
+          </tr>
+          <tr>
+            <td class="u-align--right" data-heading="Ubuntu LTS">20.04</td>
+            <td data-heading="FIPS certified component">OpenSSL, OpenSSH client, OpenSSH server</td>
+            <td data-heading="Description">General purpose cryptographic library that includes TLS implementation</td>
+            <td class="u-align--right" data-heading="Version">1:8.2p1</td>
+            <td class="u-align--right" data-heading="CMVP Certificate"><a href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/3966">3966</a></td>
+          </tr>
+          <tr>
+            <td class="u-align--right" class="u-align--right" data-heading="Ubuntu LTS">18.04</td>
+            <td data-heading="FIPS certified component">StrongSWAN</td>
+            <td data-heading="Description">IPSec based VPN solution library</td>
+            <td class="u-align--right" class="u-align--right" data-heading="Version">5.8.2</td>
+            <td class="u-align--right" class="u-align--right" data-heading="CMVP Certificate"><a href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4046">4046</a></td>
+          </tr>
+        </tbody>
+      </table>
+      <br>
+      <p><a href="https://assets.ubuntu.com/v1/8ad870d4-Ubuntu-Pro-FIPS-for-Google-Cloud-compliance-DS-14-12.22.pdf">Read the Ubuntu Pro FIPS datasheet</a></p>
+    </div>
+  </div>
+
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
+
+  <div class="row">
+    <div class="col-12">
+      <h2>Pricing</h2>
+      <table class="p-table--mobile-card">
+        <thead>
+          <tr>
+            <th style="width: 10%">Machine type</th>
+            <th>vCPUs</th>
+            <th class="u-align--right" style="width: 7%;">Memory</th>
+            <th class="u-align--right" style="width: 7%;">VM cost</th>
+            <th class="u-align--right">Ubuntu Pro cost</th>
+            <th class="u-align--right">Total cost (/hour)</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td data-heading="Machine type">n1-standard-1</td>
+            <td data-heading="vCPUs">1</td>
+            <td data-heading="Memory" class="u-align--right">3.75GB</td>
+            <td data-heading="VM cost" class="u-align--right">$0.047</td>
+            <td data-heading="Ubuntu Pro cost" class="u-align--right">$0.002</td>
+            <td data-heading="Total cost (/hour)" class="u-align--right">$0.049</td>
```

Comment:
```suggestion
            <td data-heading="Memory">3.75GB</td>
            <td data-heading="VM cost">$0.047</td>
            <td data-heading="Ubuntu Pro cost">$0.002</td>
            <td data-heading="Total cost (/hour)">$0.049</td>
```
This will fix the alignment of the numbers. Was there design input on this graph as the alignment is inconsistent?

Reply:
Well, this is coming from https://ubuntu.com/azure/fips, even content is almost the same